build_library/disk_util

```diff
@@ -351,6 +351,22 @@ def Sudo(cmd, stdout_null=False):
     if null:
       null.close()
 
+def SudoPipe(cmd):
```

Comment:
Since it is returning a string instead of a pipe perhaps name this SudoOutput instead.